Buffers are undersized for LVGL symbols on OLED screens

Warnings are shown indicating LVGL symbols can potentially cause array overflow:

```
[1/518] Preparing syscall dependency handling

[2/518] Generating include/generated/version.h
-- Zephyr version: 3.5.0 (/home/mike/compile/zmk/zephyr), build: 87fa801c5b91
[61/518] Building C object CMakeFiles/app.dir/src/display/widgets/layer_status.c.obj
In file included from /home/mike/zephyr-sdk-0.16.5-1/arm-zephyr-eabi/picolibc/include/stdio.h:435,
                 from /home/mike/compile/zmk/zephyr/include/zephyr/sys/printk.h:65,
                 from /home/mike/compile/zmk/zephyr/include/zephyr/kernel_includes.h:36,
                 from /home/mike/compile/zmk/zephyr/include/zephyr/kernel.h:17,
                 from /home/mike/compile/zmk/app/src/display/widgets/layer_status.c:7:
In function 'set_layer_symbol',
    inlined from 'layer_status_update_cb' at /home/mike/compile/zmk/app/src/display/widgets/layer_status.c:43:60,
    inlined from 'widget_layer_status_work_cb' at /home/mike/compile/zmk/app/src/display/widgets/layer_status.c:51:1:
/home/mike/compile/zmk/modules/lib/gui/lvgl/src/../src/hal/../draw/../misc/../font/lv_symbol_def.h:212:35: warning: '__builtin___sprintf_chk' may write a terminating nul past the end of the destination [-Wformat-overflow=]
  212 | #define LV_SYMBOL_KEYBOARD        "\xEF\x84\x9C" /*61724, 0xF11C*/
      |                                   ^~~~~~~~~~~~~~
/home/mike/compile/zmk/app/src/display/widgets/layer_status.c:29:23: note: in expansion of macro 'LV_SYMBOL_KEYBOARD'
   29 |         sprintf(text, LV_SYMBOL_KEYBOARD " %i", state.index);
      |                       ^~~~~~~~~~~~~~~~~~
/home/mike/compile/zmk/app/src/display/widgets/layer_status.c:29:9: note: '__builtin___sprintf_chk' output between 6 and 8 bytes into a destination of size 7
   29 |         sprintf(text, LV_SYMBOL_KEYBOARD " %i", state.index);
      |         ^~~~~~~
[69/518] Building C object CMakeFiles/app.dir/src/display/widgets/output_status.c.obj
In file included from /home/mike/zephyr-sdk-0.16.5-1/arm-zephyr-eabi/picolibc/include/stdio.h:435,
                 from /home/mike/compile/zmk/zephyr/include/zephyr/sys/printk.h:65,
                 from /home/mike/compile/zmk/zephyr/include/zephyr/kernel_includes.h:36,
                 from /home/mike/compile/zmk/zephyr/include/zephyr/kernel.h:17,
                 from /home/mike/compile/zmk/app/src/display/widgets/output_status.c:7:
In function 'set_status_symbol',
    inlined from 'output_status_update_cb' at /home/mike/compile/zmk/app/src/display/widgets/output_status.c:65:60,
    inlined from 'widget_output_status_work_cb' at /home/mike/compile/zmk/app/src/display/widgets/output_status.c:68:1:
/home/mike/compile/zmk/modules/lib/gui/lvgl/src/../src/hal/../draw/../misc/../font/lv_symbol_def.h:224:35: warning: '%i' directive output may be truncated writing between 1 and 11 bytes into a region of size 6 [-Wformat-truncation=]
  224 | #define LV_SYMBOL_WIFI            "\xEF\x87\xAB" /*61931, 0xF1EB*/
      |                                   ^~~~~~~~~~~~~~
/home/mike/compile/zmk/app/src/display/widgets/output_status.c:54:42: note: in expansion of macro 'LV_SYMBOL_WIFI'
   54 |             snprintf(text, sizeof(text), LV_SYMBOL_WIFI " %i " LV_SYMBOL_SETTINGS,
      |                                          ^~~~~~~~~~~~~~
/home/mike/compile/zmk/modules/lib/gui/lvgl/src/../src/hal/../draw/../misc/../font/lv_symbol_def.h:224:35: note: directive argument in the range [-2147483647, 2147483647]
  224 | #define LV_SYMBOL_WIFI            "\xEF\x87\xAB" /*61931, 0xF1EB*/
      |                                   ^~~~~~~~~~~~~~
/home/mike/compile/zmk/app/src/display/widgets/output_status.c:54:42: note: in expansion of macro 'LV_SYMBOL_WIFI'
   54 |             snprintf(text, sizeof(text), LV_SYMBOL_WIFI " %i " LV_SYMBOL_SETTINGS,
      |                                          ^~~~~~~~~~~~~~
/home/mike/compile/zmk/app/src/display/widgets/output_status.c:54:13: note: '__builtin_snprintf' output between 10 and 20 bytes into a destination of size 10
   54 |             snprintf(text, sizeof(text), LV_SYMBOL_WIFI " %i " LV_SYMBOL_SETTINGS,
      |             ^~~~~~~~
/home/mike/compile/zmk/modules/lib/gui/lvgl/src/../src/hal/../draw/../misc/../font/lv_symbol_def.h:224:35: warning: '%i' directive output may be truncated writing between 1 and 11 bytes into a region of size 6 [-Wformat-truncation=]
  224 | #define LV_SYMBOL_WIFI            "\xEF\x87\xAB" /*61931, 0xF1EB*/
      |                                   ^~~~~~~~~~~~~~
/home/mike/compile/zmk/app/src/display/widgets/output_status.c:50:46: note: in expansion of macro 'LV_SYMBOL_WIFI'
   50 |                 snprintf(text, sizeof(text), LV_SYMBOL_WIFI " %i " LV_SYMBOL_CLOSE,
      |                                              ^~~~~~~~~~~~~~
/home/mike/compile/zmk/modules/lib/gui/lvgl/src/../src/hal/../draw/../misc/../font/lv_symbol_def.h:224:35: note: directive argument in the range [-2147483647, 2147483647]
  224 | #define LV_SYMBOL_WIFI            "\xEF\x87\xAB" /*61931, 0xF1EB*/
      |                                   ^~~~~~~~~~~~~~
/home/mike/compile/zmk/app/src/display/widgets/output_status.c:50:46: note: in expansion of macro 'LV_SYMBOL_WIFI'
   50 |                 snprintf(text, sizeof(text), LV_SYMBOL_WIFI " %i " LV_SYMBOL_CLOSE,
      |                                              ^~~~~~~~~~~~~~
/home/mike/compile/zmk/app/src/display/widgets/output_status.c:50:17: note: '__builtin_snprintf' output between 10 and 20 bytes into a destination of size 10
   50 |                 snprintf(text, sizeof(text), LV_SYMBOL_WIFI " %i " LV_SYMBOL_CLOSE,
      |                 ^~~~~~~~
/home/mike/compile/zmk/modules/lib/gui/lvgl/src/../src/hal/../draw/../misc/../font/lv_symbol_def.h:224:35: warning: '%i' directive output may be truncated writing between 1 and 11 bytes into a region of size 6 [-Wformat-truncation=]
  224 | #define LV_SYMBOL_WIFI            "\xEF\x87\xAB" /*61931, 0xF1EB*/
      |                                   ^~~~~~~~~~~~~~
/home/mike/compile/zmk/app/src/display/widgets/output_status.c:47:46: note: in expansion of macro 'LV_SYMBOL_WIFI'
   47 |                 snprintf(text, sizeof(text), LV_SYMBOL_WIFI " %i " LV_SYMBOL_OK,
      |                                              ^~~~~~~~~~~~~~
/home/mike/compile/zmk/modules/lib/gui/lvgl/src/../src/hal/../draw/../misc/../font/lv_symbol_def.h:224:35: note: directive argument in the range [-2147483647, 2147483647]
  224 | #define LV_SYMBOL_WIFI            "\xEF\x87\xAB" /*61931, 0xF1EB*/
      |                                   ^~~~~~~~~~~~~~
/home/mike/compile/zmk/app/src/display/widgets/output_status.c:47:46: note: in expansion of macro 'LV_SYMBOL_WIFI'
   47 |                 snprintf(text, sizeof(text), LV_SYMBOL_WIFI " %i " LV_SYMBOL_OK,
      |                                              ^~~~~~~~~~~~~~
/home/mike/compile/zmk/app/src/display/widgets/output_status.c:47:17: note: '__builtin_snprintf' output between 10 and 20 bytes into a destination of size 10
   47 |                 snprintf(text, sizeof(text), LV_SYMBOL_WIFI " %i " LV_SYMBOL_OK,
      |                 ^~~~~~~~
```

## Reproduce

1. `git checkout 6946ca8b07265574a76f9ad5678fedb922756fbf`
2. Apply patch attached to this issue to add OLED screen support to the corne keyboard
[0001-add-OLED-screen-to-corne-keyboard-to-reproduce-warni.patch.txt](https://github.com/user-attachments/files/16775710/0001-add-OLED-screen-to-corne-keyboard-to-reproduce-warni.patch.txt)
3. `west build -d build/left -b nice_nano_v2 -p -- -DSHIELD=corne_left`

## Proposed Fix

I will post a PR that proposes a fix (simply increasing the array sizes used) in just a few minutes.

## Related

It's possible this will solve the second part of #808

--- a/app/src/display/widgets/layer_status.c
+++ b/app/src/display/widgets/layer_status.c
@@ -24,13 +24,13 @@ struct layer_status_state {
 
 static void set_layer_symbol(lv_obj_t *label, struct layer_status_state state) {
     if (state.label == NULL) {
-        char text[7] = {};
+        char text[8] = {};
 
-        sprintf(text, LV_SYMBOL_KEYBOARD " %i", state.index);
+        snprintf(text, sizeof(text), LV_SYMBOL_KEYBOARD " %i", state.index);
 
         lv_label_set_text(label, text);
     } else {
-        char text[13] = {};
+        char text[14] = {};
 
         snprintf(text, sizeof(text), LV_SYMBOL_KEYBOARD " %s", state.label);
 

--- a/app/src/display/widgets/output_status.c
+++ b/app/src/display/widgets/output_status.c
@@ -35,7 +35,7 @@ static struct output_status_state get_state(const zmk_event_t *_eh) {
 }
 
 static void set_status_symbol(lv_obj_t *label, struct output_status_state state) {
-    char text[10] = {};
+    char text[20] = {};
 
     switch (state.selected_endpoint.transport) {
     case ZMK_TRANSPORT_USB:

